engine.py

```diff
@@ -634,6 +635,9 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
         pm.dockControl(panel_id, area="right", content=window, label=title)
         self.log_debug("Created panel %s" % panel_id)
 
+        # Once Maya will have settled, raise the docked panel window to the top dock tab.
```

Comment:
This comment makes sense in the case when you are showing a panel at startup, but not in the case you for example are clicking the shotgun menu to launch the panel. Would be nice to tweak it to be a little more generic!